python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -594,6 +595,28 @@ def test_aggregate_e2e(
         assert row.as_pydict() == {"value": idx, "count()": 1}
 
 
+def test_plan_initial_stats(
```

Comment:
curious - is this test failing without this PR?

Reply:
you're right, the PR had no impact on the test. i replaced it with a different one